service/src/main/java/uk/gov/hmcts/reform/fpl/config/AsyncConfiguration.java

```diff
@@ -24,4 +31,26 @@ public void handleUncaughtException(Throwable throwable, Method method, Object..
         }
     }
 
+    @Override
+    @Bean
+    public Executor getAsyncExecutor() {
```

Comment:
Would it be possible to test it somehow by int tests?

Reply:
sure, will do

Reply:
done